src/Compilers/CSharp/Portable/Syntax/SyntaxExtensions.cs

```diff
@@ -30,6 +30,10 @@ internal static ArrowExpressionClauseSyntax GetExpressionBodySyntax(this CSharpS
                 case SyntaxKind.MethodDeclaration:
                     arrowExpr = ((MethodDeclarationSyntax)node).ExpressionBody;
                     break;
+                case SyntaxKind.GetAccessorDeclaration:
+                case SyntaxKind.SetAccessorDeclaration:
+                    arrowExpr = ((AccessorDeclarationSyntax)node).ExpressionBody;
+                    break;
```

Comment:
Added, also corrected `SyntaxTreeSemanticModel.cs/GetMemberModel`, because of bugs, where `GetExpressionBodySyntax` is referenced.